.github/workflows/contracts.yml

```diff
@@ -153,9 +153,22 @@ jobs:
 
       - name: Deploy contracts
         env:
-          CHAIN_API_URL: ${{ secrets.GOERLI_ETH_HOSTNAME_HTTP }}
-          CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY: ${{ secrets.GOERLI_ETH_CONTRACT_OWNER_PRIVATE_KEY }}
-          KEEP_CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY: ${{ secrets.GOERLI_KEEP_ETH_CONTRACT_OWNER_PRIVATE_KEY }}
+          # Use fake ternary expressions to decide which credentials to use,
+          # depending on chosen environment. Note: if `GOERLI...` credentials
+          # are empty, the expressions will be evaluated to the `SEPOLIA...`
+          # ones.
```

Comment:
If `environment` is not set at all (it can be as this input is not required), we also fallback to `SEPOLIA_*` right? Is this a desired behavior? If so, can we mention about it here as well?

Reply:
It makes sense to make `environment` a required input, I did it in 440c305e7f68d536b561a1fdc4091c67cd96789f. I also removed some of the ternary expressions (in e2c80e627aab48022a3fe66b1ab924ba654784d4), as we are reusing the same deployer address for Goerli and Sepolia.